DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -1,10 +1,30 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
 
 public class ComputerGeekGame
 {
+    private string _scoreBoardfile = "DeveloperGameLeaderBoard.txt";
+    private List<KeyValuePair<int, string>> _leaderBoard = new List<KeyValuePair<int, string>>();
```

Comment:
If you want to store the Name, Score, and Time, this won't work as it can only hold two things. You will probably need to create a custom class to hold all the data.

Reply:
Should I just drop the idea of the time?

Reply:
No, keep it just make another class that holds all 3 fields

Reply:
Fixed.